initializer/main.go

```diff
@@ -71,20 +77,6 @@ func main() {
 
 	flag.Parse()
 
-	testSuite := ava_testsuite.AvaTestSuite{}
```

Comment:
I intentionally put this before the loglevel-parsing, so that adding `--list` to any command line flag would print the tests regardless of what the other flags are. Any reason this is being moved?